internal/cmd/marketplace/list_versions.go

```diff
@@ -0,0 +1,130 @@
+// Copyright Mia srl
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package marketplace
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/mia-platform/miactl/internal/client"
+	"github.com/mia-platform/miactl/internal/clioptions"
+	"github.com/mia-platform/miactl/internal/resources/marketplace"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+)
+
+const listItemVersionsEndpointTemplate = "/api/backend/marketplace/tenants/%s/resources/%s/versions"
+
+var (
+	ErrItemNotFound       = errors.New("item not found")
+	ErrGenericServerError = errors.New("server error while fetching item versions")
+	ErrMissingCompanyID   = errors.New("companyID is required")
+)
+
+// ListVersionCmd return a new cobra command for listing marketplace item versions
+func ListVersionCmd(options *clioptions.CLIOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-versions",
+		Short: "List versions of a Marketplace item",
+		Long: `List the currently available versions of a Marketplace item.
+The command will output a table with each version of the item.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			restConfig, err := options.ToRESTConfig()
+			cobra.CheckErr(err)
+			client, err := client.APIClientForConfig(restConfig)
+			cobra.CheckErr(err)
+
+			releases, err := getItemVersions(
+				client,
+				restConfig.CompanyID,
+				options.MarketplaceItemID,
+			)
+			cobra.CheckErr(err)
+
+			table := buildItemVersionList(releases)
```

Comment:
```suggestion
			table := buildItmVersionListTable(releases)
```
I would suggest a method rename to clarify that it's building the table